static/js/components/ProfileDropdown.jsx

```diff
@@ -33,9 +34,23 @@ const ProfileDropdown = () => {
     <Container>
       <Dropdown ref={dropdown}>
         <DropdownTrigger>
-          { profile.username }
-          {' '}
+          <div style={{ display: "flex",
```

Comment:
This is fine for now, but I'll be writing up a styling guide soon.  I recommend we do not use inline styles in general, and instead use `makeStyles` and `useStyles`.  No reason to change this for now, I will update when I review the entire code for consistent styling.